samza-core/src/main/java/org/apache/samza/util/FutureUtil.java

```diff
@@ -143,20 +143,32 @@ public static <K, V> CompletableFuture<Map<K, V>> toFutureOfMap(
   }
 
   public static <T> CompletableFuture<T> executeAsyncWithRetries(String opName,
-      Supplier<? extends CompletionStage<T>> action,
-      Predicate<? extends Throwable> abortRetries,
+      Supplier<? extends CompletionStage<T>> action, Predicate<? extends Throwable> abortRetries,
```

Comment:
Is this method still used somewhere? If not, remove.

Reply:
Not used anymore. Left it, in case someone wanted to use the default retry policy. Removing it for now